services/command-router/src/main/java/org/eclipse/hono/commandrouter/impl/DelegatingCommandRouterAmqpEndpoint.java

```diff
@@ -200,6 +204,11 @@ protected Future<Message> processRegisterCommandConsumer(final Message request,
                 getClass().getSimpleName()
         ).start();
 
+        final Boolean sendEvent =  Optional.ofNullable(AmqpUtils.getApplicationProperty(
+                request,
+                MessageHelper.APP_PROPERTY_SEND_EVENT,
+                Boolean.class)).orElse(false);
+
```

Comment:
FMPOV we should put this into a private method returning a primitive *boolean* so that there is no need to bother if it is `null` ...

Reply:
agree, will do.